ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -205,7 +205,7 @@ private KsqlEntity executeStatement(
             || statement instanceof DropStream
             || statement instanceof DropTable
     ) {
-      //   getStatementExecutionPlan(statement, statementText, streamsProperties);
+      getStatementExecutionPlan(statement, statementText, streamsProperties);
```

Comment:
So the regression was that a statement was commented out? What was the impact exactly?

Reply:
Yes, this line evaluates the query by generating the execution plan without executing it. This will detect if the CREATE TABLE AS SELECT and CREATE STREAM AS SELECT were correct or not. Without this check the wrong queries will be sent for execution.

Reply:
@hjafarpour it doesn't use the return type, though? So are you saying that it is validating that they syntax is correct? If so, then maybe add a comment as such or have a method named `validateStatementExecutionPlan` - as it stands it looks like this method isn't doing anything

Reply:
@dguy good point. I added comment to clarify the method call.